filebeat/beater/publish.go

```diff
@@ -204,6 +217,7 @@ func (p *asyncLogPublisher) collect() bool {
 		// Tell the registrar that we've successfully sent these events
 		select {
 		case <-p.done:
+			logp.Info("Shutting down - No registrar update for successfully published batch.")
```

Comment:
we're forwarding registrar updates via queue. If queue is blocking, but stop signal has been send (p.done being closed), we do not wait for registrar to finish writing, but instead prefer to shut-down and resend the same line on next restart. This is old behavior, I just added info message notifying users on shutdown registrar will not be updated anymore.